src/templates/Profile/Portfolio/AssetsTable/index.tsx

```diff
@@ -61,53 +63,107 @@ export const AssetsTable = ({ pools }: IAssetsTableProps) => {
               </S.ChainLogoWrapper>
             </S.Imagecontainer>
             <S.FundWrapper>
-              <span>{pool.name}</span>
-              <span>{pool.symbol}</span>
+              {pool?.name ? (
+                <span>{pool.name}</span>
+              ) : (
+                <S.SkeletonLoadingWrapper>
+                  <SkeletonLoading height={1.8} width={14} />
+                </S.SkeletonLoadingWrapper>
+              )}
+              {pool?.symbol ? (
+                <span>{pool.symbol}</span>
+              ) : (
+                <S.SkeletonLoadingWrapper>
+                  <SkeletonLoading height={1.8} width={14} />
+                </S.SkeletonLoadingWrapper>
+              )}
             </S.FundWrapper>
           </S.ProductWrapper>
         </S.Td>
-        <S.Td>${parseFloat(pool.price).toFixed(2)}</S.Td>
         <S.Td>
-          ${pool.tvl ? BNtoDecimal(Big(pool?.tvl ?? Big(0)), 2) : '0'}
+          {pool?.price ? (
+            <>${parseFloat(pool.price).toFixed(2)}</>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
+        </S.Td>
+        <S.Td>
+          {pool?.tvl ? (
+            <>${pool.tvl ? BNtoDecimal(Big(pool?.tvl ?? Big(0)), 2) : '0'}</>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
         </S.Td>
         <S.Td>
-          <S.Change change={parseFloat(pool.changeMonth)}>
-            {pool.changeMonth}%
-          </S.Change>
+          {pool?.changeMonth ? (
+            <S.Change change={parseFloat(pool.changeMonth)}>
+              {pool.changeMonth}%
+            </S.Change>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
         </S.Td>
         <S.Td>
-          <S.Change change={parseFloat(pool.changeDay)}>
-            {pool.changeDay}%
-          </S.Change>
+          {pool?.changeDay ? (
+            <S.Change change={parseFloat(pool.changeDay)}>
+              {pool.changeDay}%
+            </S.Change>
+          ) : (
+            <S.SkeletonLoadingWrapper>
+              <SkeletonLoading height={1.8} width={8} />
+            </S.SkeletonLoadingWrapper>
+          )}
         </S.Td>
         <S.Td>
-          <S.FlexWrapper>
-            <div>
-              {pool.balance
-                ? BNtoDecimal(Big(pool?.balance ?? Big(0)), 18, 2)
-                : 0}{' '}
-              <span>{pool.symbol}</span>
-            </div>
-            <span>
-              $
-              {pool.balance && pool.price
-                ? BNtoDecimal(Big(pool?.balanceInUSD ?? Big(0)), 18, 2)
-                : 0}
-            </span>
-          </S.FlexWrapper>
+          {pool?.balance ? (
```

Comment:
`balance` could be `null`

```suggestion
          {pool?.balance != null ? (
```